conf/application.conf

```diff
@@ -96,11 +96,11 @@ mongodb {
   timeToLive = 60minutes
 }
 
-useBusinessDetailsStub = false
-useEncryption = false
 income-tax-submission-stub-url = "https://income-tax-submission-stub.protected.mdtp:443"
 useGetCalcListIFPlatform = true
```

Comment:
This one needs moving too to match app-config PR

Reply:
https://github.com/hmrc/income-tax-calculation/pull/110/files#diff-40b59660a23903e3ce6c42593b69e162e754b949c3fe120777db7e159a5ea529R79